.travis.yml

```diff
@@ -19,3 +19,7 @@ script:
   - diff -u <(echo -n) <(gofmt -d -s .)
   - go tool vet .
   - go test -v -race ./v3.2/...
+  - go get -t -v ./v3.3/...
+  - diff -u <(echo -n) <(gofmt -d -s .)
+  - go tool vet .
+  - go test -v -race ./v3.3/...
```

Comment:
You can merge these lines. It will become:

```yml
script:
  - go get -t -v ./v3.2/... ./v3.3/...
  - diff -u <(echo -n) <(gofmt -d -s .)
  - go tool vet .
  - go test -v -race ./v3.2/... ./v3.3/...
```